mods/d2k/rules/defaults.yaml

```diff
@@ -117,6 +117,9 @@
 		RequiresCondition: rank-elite
 		ZOffset: 256
 
+^RevealsShroudOnFire:
```

Comment:
This allows the properties of the trait (like duration) being fine-tweaked only once.